drivers/processor/processr/cpu.inf

```diff
@@ -217,11 +217,42 @@ Standard = "Πρότυπος επεξεργαστής"
 
 [Strings.0a]
 ReactOS = "Equipo de ReactOS"
+ProcessorClassName = "Procesadores"
+
+CyrixMfg = "Cyrix"
+IntelMfg = "Intel"
+AMDMfg = "Advanced Micro Devices"
+TransmetaMfg = "Transmeta"
+VIAMfg = "VIA"
```

Comment:
I'm personally not sure it's useful to "translate" the names of the vendors... when there's no translation being done!

Reply:
Ok, this could be erased. I'll fix it ASAP to don't duplicate.